src/clients/c++/library/CMakeLists.txt

```diff
@@ -148,3 +155,29 @@ install(
   DESTINATION include
 )
 
+# cmake configuration
+include (CMakePackageConfigHelpers)
+set (_LIB_CMAKE_DIR lib/cmake/Trtis)
+install(
+  EXPORT trtis-export
+  FILE TrtisTargets.cmake
+  NAMESPACE Trtis::
```

Comment:
TRTIS::
and similar changes below for Trtis -> TRTIS

Reply:
All done.